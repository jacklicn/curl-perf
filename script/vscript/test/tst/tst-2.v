var i,b

i = 0
b = 2
while i < 3
   i = i + 1
   b = b * 2
end


print(i)
print(b)
