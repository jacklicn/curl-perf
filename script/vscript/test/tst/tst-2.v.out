*** syntax ok ***
*** semantics and code generation ok ***

>3

>16
